source/documentation/developing-now.html.md.erb

```diff
@@ -12,7 +12,7 @@ weight: 3
 
 | **Project** |**Description** |
 |------|-------------|
-|**Northern Ireland**| We will develop the CTC API functionality so that it can manage transit to and from Northern Ireland after the end of the transition period. This functionality is due to be released by the end of March 2021.  
+|**Northern Ireland**| The CTC API functionality will manage transit to and from Northern Ireland. We expect this functionality to be available for you to test with Trader Test in April 2021. We will release the same functionality to production soon after.  
```

Comment:
The Dev Hub refers to the "Sandbox environment" not "Trader Test" since TT is only part of the Sandbox.